src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -286,7 +286,9 @@ public String toString() {
     KEY_STORE_SECRET("keyStoreSecret", ""),
     KEY_STORE_LOCATION("keyStoreLocation", ""),
     SSL_PROTOCOL("sslProtocol", SSLProtocol.TLS.toString()),
-    MSI_CLIENT_ID("msiClientId", ""),;
+    MSI_CLIENT_ID("msiClientId", ""),
+    KEY_VAULT_COLUMN_ENCRYPTION_PROVIDER_CLIENT_ID("keyVaultColumnEncryptionProviderClientId", ""),
```

Comment:
Since the property names is seeming very long, we would propose renaming properties to under:
`keyVaultProviderClientId` and `keyVaultProviderClientKey`

Reply:
Cool, wasn't sure if there were any naming standards so I'm happy with whatever Microsoft want to go with.